### PR TITLE
Added restart button with functionality.

### PR DESCRIPTION
**Pull request by Tursunpulat2002**
The restart button refreshes the timer, the word bank, and the input field.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -20,28 +31,50 @@ function renderWords() {
...
         spanElement.setAttribute("id", i);
+        spanElement.setAttribute("class", "fs-4");
         spanElement.innerHTML = game.words[i];
         document.getElementById("random-words").appendChild(spanElement);
     }
 }
 
+function derenderWords() {
+    for (let i = 0; i < game.words.length; i++) {
+        const item = document.getElementById(i);
```
Comment (omonkulov) [inline index.js]: Missing comments. What is the purpose of this function?

File: index.js [hunk 2]
```diff
@@ -20,28 +31,50 @@ function renderWords() {
...
-    correctWordElement.setAttribute("class", "text-success");
+    correctWordElement.setAttribute("class", "text-success fs-4");
     game.correctWords += 1;
     game.currentWord += 1;
 }
 
 function renderErrorWord(wordID) {
     let errorWordElement = document.getElementById(wordID);
-    errorWordElement.setAttribute("class", "text-danger");
+    errorWordElement.setAttribute("class", "text-danger fs-4");
```
Comment (omonkulov) [inline index.js:68]: What is fs-4?

File: index.js [hunk 3]
```diff
@@ -1,12 +1,23 @@
 import passagesArray from "./src/passages.js";
 import game from "./src/game_state.js";
 
+let intervalID = null;
```
Comment (omonkulov) [inline index.js:5]: What is this variable used for?

**Comment by omonkulov** [comment]
Use this pull request to properly document everything.

File: index.js (hunk 1 above)
Comment (Tursunpulat2002) [reply index.js]: Updated.

File: index.js (hunk 2 above)
Comment (Tursunpulat2002) [reply index.js:68]: Updated.

File: index.js (hunk 3 above)
Comment (Tursunpulat2002) [reply index.js:5]: Updated

**Comment by Tursunpulat2002** [comment]
> Use this pull request to properly document everything.

Finished with the new commit

**Review by omonkulov — APPROVED** [review]
Looks great to me

**Pull request merged**